test/upgradable/Upgradable.js

```diff
@@ -26,9 +26,8 @@ describe('Upgradable', () => {
         upgradableTestFactory = await ethers.getContractFactory('UpgradableTest', ownerWallet);
 
         create3DeployerFactory = await ethers.getContractFactory('Create3Deployer', ownerWallet);
-    });
 
-    beforeEach(async () => {
```

Comment:
let's avoid reusing the contract instance for positive tests unless the tests are really independent (unlike in this case). you can group revert tests and have a single before block for it. for positive tests, can still have a before each

Reply:
Sounds good!